### ISSUE TITLE
Engine refers to global Entity

### ISSUE THREAD

**Issue by rosshadden** [lovetoys/lovetoys#53]
It looks like [engine still refers to the `Entity` global](https://github.com/lovetoys/lovetoys/blob/49d07cdd57a382002874e77484e019db442e3ce4/src/Engine.lua#L9), which causes it to crash with:

```
Error: vendor/lovetoys/src/Engine.lua:9: attempt to call global 'Entity' (a nil value)
```

**Comment by rosshadden** [comment]
And `EventManager` on line `9`.

**Comment by raffomania** [comment]
Whoops. I'm pushing the fix in a minute. I can't find the error in `EventManager` though, could you comment with a link to the exact line?

**Comment by rosshadden** [comment]
Oh lol, I fixed it in #54

**Comment by raffomania** [comment]
Haha, perfect. I'll merge it after the tests pass. Thanks!
